src/components/LoginsPage.tsx

```diff
@@ -0,0 +1,175 @@
+import Head from "next/head";
```

Comment:
This is not a page. You could name this `Identities`, and `src/pages/account/logins/index.tsx` should be called `LoginsPage`.

Reply:
OK fixed ce50f4b